eth/downloader/downloader.go

```diff
@@ -229,7 +229,7 @@ func New(checkpoint uint64, stateDb ethdb.Database, mux *event.TypeMux, chain Bl
 		dropPeer:       dropPeer,
 		headerProcCh:   make(chan *headerTask, 1),
 		quitCh:         make(chan struct{}),
-		SnapSyncer:     snap.NewSyncer(stateDb, chain.TrieDB().Scheme()),
+		SnapSyncer:     snap.NewSyncer(stateDb, rawdb.HashScheme),
```

Comment:
Let's use `chain.TrieDB().Scheme()`. The hardcoded string might be forgot to update in the future.

Reply:
Yep, this was a change that I made before I changed the `TrieDB().Scheme()` return, then forgot to undo it